cmd/earthly/secret_cmds.go

```diff
@@ -291,7 +291,12 @@ func (app *earthlyApp) actionSecretsListV2(cliCtx *cli.Context) error {
 	}
 
 	for _, secret := range secrets {
-		fmt.Println(secret.Path)
+		display := secret.Path
+		if !strings.HasPrefix(display, "/user") {
+			prefix := fmt.Sprintf("/%s/%s/", cliCtx.String("org"), cliCtx.String("project"))
```

Comment:
Can you use the variables here, rather than `cliCtx.String(...)`?

Reply:
Which variables?

Reply:
That long list that you don't really like :-)

https://github.com/earthly/earthly/blob/main/cmd/earthly/main.go#L65

Reply:
I was going to PR the `--project` flag (rather than arg) changes separately. Mind if we switch to the `cliFlags` struct there?

Reply:
:+1: